projects/shared/src/lib/component/documents/files/files.component.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019-2023 RERO
+ * Copyright (C) 2019-2023 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { HttpClient } from "@angular/common/http";
+import { Component, Input, OnInit, inject } from "@angular/core";
+import { ApiService } from "@rero/ng-core";
+import { Record, RecordService } from "@rero/ng-core";
+import { PrimeNGConfig } from "primeng/api";
+import { forkJoin, map, switchMap } from "rxjs";
+
+// file interface
+export interface File {
+  // thumbnail URL
+  thumbnail: string;
+  // downloand URL
+  download: string;
+  // thumbnail legend
+  label: string;
+}
+
+@Component({
+  selector: "shared-doc-files",
+  templateUrl: "./files.component.html",
+  styleUrl: "./files.component.scss",
+})
+export class FilesComponent implements OnInit {
+
+  // input document pid
+  @Input() documentPid: string;
+
+  // list of files
+  files : Array<File> = [];
```

Comment:
`File[]`

Reply:
Both are similar. Which one is better? Any recommandation?

Reply:
Angular code uses `File[]`. This is the syntax we had chosen to pursue the devs 😉